README.md

```diff
@@ -20,6 +20,14 @@ $ ./simulator
 Modbus TCP address: 0.0.0.0:1502 
 Start up a Modbus TCP simulator.
 ```
+### Running the device-modbus Service as a Snap
+EdgeX Device modbus is also available as a snap package. Install the snap with the following command:
+```
+sudo snap install edgex-device-modbus
+```
+for more details on the Device modbus Snap, including installation, configuration, please refer to [EdgeX Modbus Device Service Snap](https://github.com/edgexfoundry/device-modbus-go/tree/main/snap)
+
+for more details on Snap, including EdgeX Snap, viewing logs, security services, please check [Getting Started with Snap](https://docs.edgexfoundry.org/2.0/getting-started/Ch-GettingStartedSnapUsers/)
```

Comment:
2x ~for~ For 

more details on ~Snap, including EdgeX Snap~ deploying EdgeX with Snaps, viewing logs

Reply:
Updated. Thanks!